public/styles/components/_datepicker.css

```diff
@@ -0,0 +1,744 @@
+.CalendarDay {
```

Comment:
Yep, this file actually is unnecessary (I forgot to delete it before pushing it) as the components is loading its  own scss. The style customisation is done by overriding the variables in the _variables.scss file.